src/charm.py

```diff
@@ -303,6 +302,19 @@ def logs_rules(self) -> Dict[str, Any]:
         """Return a list of logging rules."""
         return self._cos.logs_alerts
 
+    @property
+    def requested_receivers(self) -> set:
+        """Return a list of requested tracing receivers."""
+        protocols = self._cos.requested_protocols()
+        protocols.update(
+            [
+                receiver
+                for receiver in get_args(ReceiverProtocol)
```

Comment:
neat. I never use get_args but it's handy. Wonder if the whole thing would have been cleaner had ReceiverProtocol been an enum... 
Food for thought.

Reply:
It could be a bit cleaner, however then it'd be useful to do it on tempo side as well not to have inconsistencies. I'd say we keep it for improvement ideas for later.